site-manage/site-group-manager/src/main/java/org/sakaiproject/groupmanager/controller/MainController.java

```diff
@@ -68,8 +68,7 @@ public class MainController {
     public String showIndex(Model model, HttpServletRequest request, HttpServletResponse response) {
         log.debug("showIndex()");
         
-        String userId = sessionManager.getCurrentSessionUserId();
-        final Locale locale = StringUtils.isNotBlank(userId) ? preferencesService.getLocale(userId) : Locale.getDefault();
```

Comment:
Do you know if "private PreferencesService preferencesService;" can be removed and cleanup?

Reply:
Yes, in both cases those variables could have been removed, so I did it. Thanks!

Reply:
Nice cleanup!